src/stores/RoomListStore.tsx

```diff
@@ -0,0 +1,77 @@
+import { makeAutoObservable, makeObservable } from "mobx";
+import { RootStore } from "@/stores/RootStore";
+import RoomService from "@/service/room.service";
+import { DEFAULT_PASSWORD, DEFAULT_THUMBNAIL } from "@/constants/default";
+import roomService from "@/service/room.service";
+import { RoomOverview } from "@/models/room/RoomOverview";
+
+export class Room {
+  readonly _id: string = "";
+  readonly _master_id: string = "test";
+  readonly _title: string = "test";
+  readonly _thumbnail?: string = "test";
+  readonly _password?: string = "test";
+  readonly _timer: number = 40;
+  readonly _short_break: number = 15;
+  readonly _long_break: number = 20;
+  readonly _long_break_interval: number = 3;
+  readonly _expired_at: string = "2021-08-21T12:30:00.000Z";
+  readonly _room_tag: string[] = [];
```

Comment:
클래스 필드 컨벤션이 다르네요. 컨벤션에 맞게 수정해주세요. 그리고 임시로 사용한다면 추후에 수정 될 사항에 대해 TODO를 작성해주세요.
```suggestion
  readonly expiredAt: string = "2021-08-21T12:30:00.000Z";
  readonly tags: string[] = [];
```